editor.planx.uk/src/hooks/useSearch.ts

```diff
@@ -61,7 +63,20 @@ export const useSearch = <T extends object>({
         };
       }),
     );
-  }, [pattern, fuse]);
+  }, [pattern, fuse, searchType]);
 
   return { results, search: setPattern };
 };
+
+const formatSearchPattern = (pattern: string) => {
+
+  // split pattern by words and remove empty spaces
+  const patternArray = pattern.split(" ").filter(Boolean)
+
+  // In FuseJS extended search the character: ' 
+  // is used for the include-match search option
+  // so it will include this whole word in matching
+  const formattedPattern = `'${patternArray.join(" '")}`
+
+  return formattedPattern
+}
```

Comment:
nit: I'd try to find a way to reduce comments like this - the cognitive load feels quite high for a simple function. I'm having to to read the js, match this to comments, and refer back to fusejs docs to really get a hold of what's happening here.

Something like this illustrates my point (there's a lot of room for improving variable names here! 😅) - 

```suggestion
const convertToIncludeMatchPattern = (fuzzyMatchPattern: string) => {
  const words = fuzzyMatchPattern.split(" ").filter(Boolean);
  // Docs: http://fuse.js/link-to-docs
  const includeCharacter = "'"
  const includeMatchPattern = words.map((word) =>`${includeCharacter}word`).join(" ");
  
  return includeMatchPattern;
}
```

This is untested code, and we're maybe drifting into personal styles / opinions here, but I think the above is more explicit and there's less room for drift between comments and code.